frontend/projects/valtimo/case/src/lib/case-inspection/tabs/logs-tab.component.scss

```
@@ -0,0 +1,172 @@
+  &__detail-timestamp {
+    color: var(--cds-text-secondary, #525252);
+    font-size: 0.75rem;
+  }
+
+  &__detail-message {
+    margin: 0.75rem 0;
+    padding: 0.5rem;
+    background-color: var(--cds-background, #ffffff);
+    border: 1px solid var(--cds-border-subtle, #e0e0e0);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)